pallets/automation-time/src/migrations/update_xcmp_task.rs

```diff
@@ -19,7 +19,7 @@ use crate::migrations::utils::{
 	OldAccountTaskId, OldAction, OldTask, OldTaskId, TEST_TASKID1,
 };
 
-const EXECUTION_FEE_AMOUNT: u128 = 3_000_000_000;
+const EXECUTION_FEE_AMOUNT: u128 = 4_000_000_000;
```

Comment:
Good call Vinh. The only way to pick it up is to run those XCM transactions in production regularly, so I think we need daily test monitoring and reporting.

Reply:
The migration only needs to be executed once, and the fee_per_second remains relatively stable in mangata-rococo and mangata-kusama. So, there is no need to worry.

Reply:
Right, normally other parachains don’t change those values, but when they do, the increment is likely to break existing tasks since the new fees will exceed the limits set in task. Down the road we will need to find a way to monitor those fee related values.